hs/Scripts.hs

```diff
@@ -44,4 +44,8 @@ timetableScripts = do jQuery
                                                 "static/js/grid/generate_grid.js",
                                                 "static/js/common/objects/course.js",
                                                 "static/js/common/objects/section.js",
-                                                "static/js/common/utilities/util.js"])
\ No newline at end of file
+                                                "static/js/common/utilities/util.js"])
+postScripts :: H.Html
+postScripts = concatHtml (map makeScript ["http://code.jquery.com/jquery-1.10.2.js",
```

Comment:
@shaftoes changed the URL for the jquery import, which you should follow here. You can find it in #303